testing/tests/block_fragments.rs

```diff
@@ -11,7 +11,7 @@ struct FragmentSimple<'a> {
 fn test_fragment_simple() {
     let simple = FragmentSimple { name: "world" };
 
-    assert_eq!(simple.render().unwrap(), "\n\n<p>Hello world!</p>\n");
+    assert_eq!(simple.render().unwrap(), "\n<p>Hello world!</p>");
```

Comment:
For all lines removal, I'm not sure whether it was bad before and this PR fixed it as a side effect or the opposite...

Reply:
Looking at this particular case, IMO the previous state was good and this PR breaks it.

```
{% block body %}
<p>Hello {{ name }}!</p>
{% endblock %}
```

Arguably this should render newlines before and after the `p` node?